### ISSUE TITLE
does it not work on an already modded dlss fg

### ISSUE THREAD

**Issue by Rigel07** [Nukem9/dlssg-to-fsr3#217]
i tried this mod on top of PureDark's DLSS 3 FG mod for GTA V. The mod works for 40 series cards but when I tried replacing it with this mod, i didnt see any popup that the mod is enabled and hence no frame gen option in the game. HAGS is enabled in windows settings and mod was installed as per the instructions. (also wanted to ask if it has a different approach to install for optimus laptops cuz im running a laptop 3050)

**Comment by Igihara** [comment]
Obviously.

**Comment by TimmyP777** [comment]
Itll get there.

**Comment by ImASadBoy** [comment]
It does work I'm currently using it, refer to #131 there's a whole thread there on how to do it

**Comment by Rigel07** [comment]
> It does work I'm currently using it, refer to #131 there's a whole thread there on how to do it

I see. Actually this mod has all the dlss dll files and stuff in a subfolder (meaning the game exe and dlss files arent together in a single folder). Is this the reason it isnt working?

**Comment by ImASadBoy** [comment]
> > It does work I'm currently using it, refer to #131 there's a whole thread there on how to do it
> 
> I see. Actually this mod has all the dlss dll files and stuff in a subfolder (meaning the game exe and dlss files arent together in a single folder). Is this the reason it isnt working?

I don't know, maybe it could be rockstar launcher maybe the wrong folder, play around and test for yourself!!

**Comment by Rigel07** [comment]
> > > It does work I'm currently using it, refer to #131 there's a whole thread there on how to do it
> > 
> > 
> > I see. Actually this mod has all the dlss dll files and stuff in a subfolder (meaning the game exe and dlss files arent together in a single folder). Is this the reason it isnt working?
> 
> I don't know, maybe it could be rockstar launcher maybe the wrong folder, play around and test for yourself!!

Btw i just saw that for elden ring too the dlss fg files are in a mod folder and not with the exe so have you put the fsr files with dlss ones in mod folder or with the exe?

**Comment by TimmyP777** [comment]
Yeah, works great in ER.

I would guess it works in The Last of Us also, but I cant get the stupid mod to validate his stupid DRM. It does load though.

**Comment by ImASadBoy** [comment]
> > > > It does work I'm currently using it, refer to #131 there's a whole thread there on how to do it
> > > 
> > > 
> > > I see. Actually this mod has all the dlss dll files and stuff in a subfolder (meaning the game exe and dlss files arent together in a single folder). Is this the reason it isnt working?
> > 
> > 
> > I don't know, maybe it could be rockstar launcher maybe the wrong folder, play around and test for yourself!!
> 
> Btw i just saw that for elden ring too the dlss fg files are in a mod folder and not with the exe so have you put the fsr files with dlss ones in mod folder or with the exe?

On #131 I posted a screenshot where the files are in the exe folder, just check the thread. It doesn't matter where the dlss files are, this Nukem9 mod requires you to be extracted in the game's exe folder.

**Comment by Rigel07** [comment]
> Yeah, works great in ER.
> 
> I would guess it works in The Last of Us also, but I cant get the stupid mod to validate his stupid DRM. It does load though.

Dang. Dk whats wrong with mine then.. tried putting it into mods folder tried putting it with exe yet no success

**Comment by TimmyP777** [comment]
Nothing goes in the mods folder, just everything into the same folder as the ER exe.

**Comment by Rigel07** [comment]
> > > > > It does work I'm currently using it, refer to #131 there's a whole thread there on how to do it
> > > > 
> > > > 
> > > > I see. Actually this mod has all the dlss dll files and stuff in a subfolder (meaning the game exe and dlss files arent together in a single folder). Is this the reason it isnt working?
> > > 
> > > 
> > > I don't know, maybe it could be rockstar launcher maybe the wrong folder, play around and test for yourself!!
> > 
> > 
> > Btw i just saw that for elden ring too the dlss fg files are in a mod folder and not with the exe so have you put the fsr files with dlss ones in mod folder or with the exe?
> 
> On #131 I posted a screenshot where the files are in the exe folder, just check the thread. It doesn't matter where the dlss files are, this Nukem9 mod requires you to be extracted in the game's exe folder.

Yep saw your screenshot after which thought of doing it that way but still same results. Even the comfirmation popup doesnt appear. Also do you have a pc or a laptop? Im a bit paranoid and hence wondering if the signature registries are different for laptop gpus

**Comment by Rigel07** [comment]
> Nothing goes in the mods folder, just everything into the same folder as the ER exe.

Ah alright. Thanks

**Comment by MinaAlmighty** [comment]
I could make the mod show me the frame generation option with RDR2 but the game crashes with Error GFX as soon as the scene renders.

**Comment by TimmyP777** [comment]
I did have ONE crash when I loaded first frame the first time. So don't only try it once.

Ive tried and tested it a dozen times since, and have not seen any crashes. Just that one time.

PS: It crashed to a game error dialog, not a Windows one.

**Comment by Rigel07** [comment]
> I could make the mod show me the frame generation option with RDR2 but the game crashes with Error GFX as soon as the scene renders.

Nah im talking about GTA not RDR2, and theres no option in settings. Frame generation toggle is in the reshade overlay which pops after pressing Home/Insert

**Comment by CasualDev242** [comment]
> > I could make the mod show me the frame generation option with RDR2 but the game crashes with Error GFX as soon as the scene renders.
> 
> Nah im talking about GTA not RDR2, and theres no option in settings. Frame generation toggle is in the reshade overlay which pops after pressing Home/Insert

It won't work because FSR3 is DX12 only and GTA5 is DX11.
